readthedocs/search/parse_json.py

```diff
@@ -35,13 +24,14 @@ def generate_sections_from_pyquery(body):
             if next_p[0].tag == 'div' and 'class' in next_p[0].attrib:
                 if 'section' in next_p[0].attrib['class']:
                     break
-            h1_content += '\n%s\n' % next_p.html()
+
+            h1_content += parse_content(next_p.text())
             next_p = next_p.next()
         if h1_content:
             yield {
                 'id': h1_id,
                 'title': h1_title,
-                'content': h1_content,
+                'content': h1_content.replace('\n', '. '),
```

Comment:
I think we can end up with duplicate `.`

Reply:
There were still new lines characters -- so I did a `replace` here.

Reply:
Looks like we should fix `parse_content` then (if replacing new lines is the job of parse_content)